rayon-core/src/sleep/mod.rs

```diff
@@ -1,15 +1,32 @@
 //! Code that decides when workers should go to sleep. See README.md
 //! for an overview.
 
+use DeadlockHandler;
```

Comment:
We should document the changes here in [the README.md file for this module](https://github.com/Zoxc/rayon/blob/f541d2fa75cad1731d13d0e84095fcf2f05bef32/rayon-core/src/sleep/README.md)

Reply:
It seems like it could just go in a little section at the end

Reply:
I added a section on deadlock detection: https://github.com/Zoxc/rayon/blob/rustc/rayon-core/src/sleep/README.md#deadlock-detection